app/models/card.rb

```diff
@@ -10,6 +10,11 @@ class Card < ActiveRecord::Base
   before_validation :define_review_date
   validates :original_text, :translated_text, presence: true
   validates_with CompareValidator
+  scope :time_to_check_card, -> { where("review_date < ?", Time.zone.now).order("RANDOM()")}
+
+  def check_translation(user_input)
+    return false ? user_input == self.translated_text : self.define_review_date && self.save
```

Comment:
Line is too long. [92/80]<br>Redundant `return` detected.<br>Redundant `self` detected.